src/box/session.h

```diff
@@ -331,6 +340,27 @@ session_remove_stmt_id(struct session *session, uint32_t stmt_id);
 void
 session_destroy(struct session *);
 
+/**
+ * Set session peer address.
+ */
+static inline void
+session_set_peer_addr(struct session *session,
+		      const struct sockaddr *addr, socklen_t addrlen)
+{
+	assert(addrlen <= sizeof(session->meta.peer.addrstorage));
+	memcpy(&session->meta.peer.addrstorage, addr, addrlen);
+	session->meta.peer.addr =
+		(struct sockaddr *)&session->meta.peer.addrstorage;
+	session->meta.peer.addrlen = addrlen;
+}
+
+/**
+ * Return session peer name or NULL if the session is local.
+ * The string is allocated in the static buffer.
+ */
+const char *
+session_peer(struct session *session);
```

Comment:
I suggest to make it `const struct session *`.

Reply:
Done.